src/tesla_ver/data_uploading/data_uploading.py

```diff
@@ -126,20 +127,31 @@ def push_to_redis(button_clicks, data_dict, selected_columns, selected_rows):
 
         serialization_context = pa.default_serialization_context()
 
+
+        # Gets session UUID to write user specific data
+        session_uuid = session.get('uuid')
+
         redis_manager.redis.set(
-            "data_numeric",
+            session_uuid + "_numeric_data",
             serialization_context.serialize(df[sorted(set(["Year", "Subject", *selected_columns]))])
             .to_buffer()
             .to_pybytes(),
         )
-        server.logger.debug("redis numeric data set")
+        server.logger.debug("redis numeric data set at key: " + session_uuid + "_numeric_data")
+
+        # Metadata storage feature flag
+
+        store_mdata = False
+
+        if store_mdata:
```

Comment:
this code will never run, was that intentional?
Under what circumstances would you want store_mdata to be True?

Reply:
I wanted to make sure that the metadata selection system remained there so that it could be used in the future to add metadata annotations, even if it's not being used right now.  That's why I have the feature flag right there, so it doesn't write unnecessary values to the database while it's not needed, but it can be really easily reenabled if/when future versions use metadata for their analyses/visualizations

Reply:
makes sense, sounds like a `#TODO:` comment is warranted